pkg/backend/display/progress.go

```diff
@@ -776,11 +776,22 @@ func (display *ProgressDisplay) printPolicyViolations() bool {
 			eventJ.PolicyPackName); packNameCmp != 0 {
 			return packNameCmp < 0
 		}
-		if packVerCmp := strings.Compare(
-			eventI.PolicyPackVersion,
-			eventJ.PolicyPackVersion); packVerCmp != 0 {
-			return packVerCmp < 0
+		if eventI.PolicyPackVersionTag != "" && eventJ.PolicyPackVersionTag != "" {
```

Comment:
Nit: You might want to consider something like introducing `iTag, jTag :=  eventI.PolicyPackVersionTag, eventJ.PolicyPackVersionTag`. Since that might allow you to put avoid splitting the code across multiple lines without sacrificing much in terms of readability.

Reply:
Going to use version now, so this new check is not longer needed.